### DISCUSSION TITLE
I really need help understanding either the script or the AO3 website.

### DISCUSSION THREAD

**Discussion by audiogamer22** [nianeyna/ao3downloader#179] [Q&A] [answered]
So I've been using this script on and off for a while and it works fine but I'm very confused.

I like Harry potter bashing so I go to this page: https://archiveofourown.org/works?work_search%5Bsort_column%5D=revised_at&include_work_search%5Bfandom_ids%5D%5B%5D=136512&work_search%5Bother_tag_names%5D=&work_search%5Bexcluded_tag_names%5D=&work_search%5Bcrossover%5D=&work_search%5Bcomplete%5D=&work_search%5Bwords_from%5D=&work_search%5Bwords_to%5D=&work_search%5Bdate_from%5D=&work_search%5Bdate_to%5D=&work_search%5Bquery%5D=&work_search%5Blanguage_id%5D=en&commit=Sort+and+Filter&tag_id=Albus+Dumbledore+Bashing

I'm blind so press h twice from the top of the page and the screen reader jumps me to a part that says 1 - 20 of 6,611 Works in Albus Dumbledore Bashing, that sounds about right, I then press b to go to include fandoms and press enter to expand the thing, then I press x to go to harry potter, and here's what it says:  
Harry Potter - J. K. Rowling (6602) then I press enter on sort and filter, but when I look at the page again, I get the same number, 1 - 20 of 6,611 Works in Albus Dumbledore Bashing.

Perhaps it helps to say i'm not loged in to ao3, i've also selected English as my language but I'm so confused. How can there be more than what is mentioned when I tick the include harry potter tickbox?

I've realised I have to do this otherwise the script downloads every single work, including non harry potter related things, is this a bug? i'd assume if I was on a harry potter page and didn't modify it at all; it would download only harry potter stuff, or is that not how it works?

**Comment by FrancaBaars** [comment, marked as answer]
I think it is just the way AO3 works. When I look at the page with only Harry Potter - J.K. Rowling tag of the fandom. It says a total of 554862 works were found, but when I then look at the filtering of the fandom it says Harry Potter - J.K. Rowling (540220). https://archiveofourown.org/tags/Harry%20Potter%20-%20J*d*%20K*d*%20Rowling/works 

If I then press on Harry Potter - J.K. Rowling again I do find that there are some subtags from this fandom tag. https://archiveofourown.org/tags/Harry%20Potter%20-%20J*d*%20K*d*%20Rowling
- LEGO Harry Potter (Video Games)
- Fantastic Beast and Where to Find Them (Movies)
- Harry Potter and the Cursed Child - Thorne & Rowling
- Harry Potter: Hogwart Mystery (Video Game)

Maybe it automatically also shows the works of these subtags when filtering works but not in the filtering menu? I don't really know but it looks like just maybe a bug or feature of AO3

**Reply by audiogamer22** [reply]
Ah ok, thanks for letting me know; I'll close this issue.
